core-models/src/main/scala/com/mesosphere/usi/core/models/commands/SchedulerCommand.scala

```diff
@@ -28,8 +29,10 @@ sealed trait SchedulerCommand
   *
   * @param podId
   * @param runSpec
+  * @param domainFilter An optional filter for default domains. See [[DomainFilter]] for details.
   */
-case class LaunchPod(podId: PodId, runSpec: RunTemplate) extends SchedulerCommand
+case class LaunchPod(podId: PodId, runSpec: RunTemplate, domainFilter: Option[DomainFilter] = None)
```

Comment:
I'd like to discuss this please; what are your thoughts on defaulting it to None vs just setting it HomeRegionFilter?

Reply:
I did not give it much thought yesterday. `HomeRegionFilter` is much better here.